addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1260,7 +1260,7 @@ options.registry.OptionsTab = options.Class.extend({
      */
     async _computeWidgetState(methodName, params) {
         if (methodName === 'customizeBodyBgType') {
-            const bgImage = $('#wrapwrap').css('background-image');
+            const bgImage = getComputedStyle(this.$target[0].ownerDocument.querySelector('#wrapwrap'))['background-image'];
```

Comment:
Don't you have `this.ownerDocument` directly? Would be visually better especially since `.$target` in the case of those option does not mean anything.